great_expectations/datasource/fluent/spark_file_path_datasource.py

```diff
@@ -36,9 +36,36 @@ def _get_reader_options_include(self) -> set[str] | None:
         return {"header", "infer_schema"}
 
 
+class ParquetAsset(_FilePathDataAsset):
+    type: Literal["parquet"] = "parquet"
+    # The options below are available for parquet as of spark v3.4.0
+    # See https://spark.apache.org/docs/latest/sql-data-sources-parquet.html for more info.
+    datetime_rebase_mode: Literal["EXCEPTION", "CORRECTED", "LEGACY"] = Field(
+        alias="datetimeRebaseMode"
+    )
+    int_96_rebase_mode: Literal["EXCEPTION", "CORRECTED", "LEGACY"] = Field(
+        alias="int96RebaseMode"
+    )
+    merge_schema: bool = Field(False, alias="mergeSchema")
+
+    class Config:
+        extra = pydantic.Extra.forbid
+        allow_population_by_field_name = True
+
+    def _get_reader_method(self) -> str:
+        return self.type
+
+    def _get_reader_options_include(self) -> set[str] | None:
+        """These options are available as of spark v3.4.0
+
+        See https://spark.apache.org/docs/latest/sql-data-sources-parquet.html for more info.
+        """
+        return {"datetimeRebaseMode", "int96RebaseMode", "mergeSchema"}
```

Comment:
This there a way to introspect the pyspark version and get this? Our min version is 2.3.2. Or maybe this doesn't matter since this is only used internally?

Reply:
It's a good point, probably. I thought for this near term effort (spark) we were going to manually create these assets since there are relatively few.